modules/@apostrophecms/module/index.js

```diff
@@ -272,7 +272,7 @@ module.exports = {
       // be used very sparingly, and pretty much never in npm modules. The
       // only exceptions in apostrophe core are `apos.area` and `apos.singleton`.
       //
-      // The helper must be added first with `addHelpers` or `addHelper`.
+      // The template helper must be added first. [See the module configuration reference](https://a3.docs.apostrophecms.org/reference/module-api/module-overview.html#customization-functions) for more.
```

Comment:
Note that we're not definitely going to be generating documentation from this. I'm hedging by using markdown syntax and using the full URL.